.github/workflows/publish-command.yml

```diff
@@ -0,0 +1,64 @@
+name: integration-test
+on:
+  workflow_dispatch:
+    inputs:
+      connector:
+        description: 'Airbyte Connector'
+        required: true
+      comment-id:
+        description: 'The comment-id of the slash command. Used to update the comment with the status.'
+        required: false
+
+jobs:
+  integration_test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Search for valid integration name format
+        id: regex
+        uses: AsasInnab/regex-action@v1
+        with:
+          regex_pattern: '^(connectors|bases)/[a-zA-Z0-9-_]+$'
+          regex_flags: 'i' # required to be set for this plugin
+          search_string: ${{ github.event.inputs.connector }}
+      - name: Validate input workflow format
+        if: steps.regex.outputs.first_match != github.event.inputs.connector
+        run: echo "The connector provided has an invalid format!" && exit 1
+      - name: Link comment to workflow run
+        if: github.event.inputs.comment-id
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          comment-id: ${{ github.event.inputs.comment-id }}
+          body: |
+            > :clock2: ${{github.event.inputs.connector}} https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}
+      - name: Checkout Airbyte
+        uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '14'
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '14.7'
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+      - run: |
+          docker login -u airbytebot -p ${DOCKER_PASSWORD}
+          SKIP_TESTS=true ./tools/integrations/manage.sh publish airbyte-integrations/${{ github.event.inputs.connector }}
```

Comment:
do we have a guarantee that integration tests have passed before running publish and skipping tests?